roles/pulp_webserver/tasks/apache.yml

```diff
@@ -22,9 +22,20 @@
           dest: /etc/apache2/sites-enabled/pulp-vhost.conf
         - src: ../mods-available/proxy.load
           dest: /etc/apache2/mods-enabled/proxy.load
+        - src: ../mods-available/proxy_http.load
+          dest: /etc/apache2/mods-enabled/proxy_http.load
+        - src: ../mods-available/ssl.load
+          dest: /etc/apache2/mods-enabled/ssl.load
+        - src: ../mods-available/headers.load
+          dest: /etc/apache2/mods-enabled/headers.load
       when: ansible_facts.os_family == "Debian"
       notify: reload apache
 
+    - name: Remove default apache vhost file
+      file:
+        state: absent
+        path: /etc/apache2/sites-available/000-default.conf
+
```

Comment:
This is under sites-available, not sites-enabled. Perhaps you meant to remove it from sites-enabled?